integration-app/src/main/java/uk/gov/crowncommercial/esourcing/auth/AuthConfiguration.java

```diff
@@ -47,12 +51,23 @@ protected void configure(HttpSecurity http) throws Exception {
           .sessionCreationPolicy(SessionCreationPolicy.STATELESS);
     // @formatter:on
   }
-  
+
 
   @Bean
   IpAddressFilter ipAddressFilter() {
-    return new IpAddressFilter(); 
+    return new IpAddressFilter(ipAllowList, null, Arrays.asList("/actuator/**"));
```

Comment:
Should we add openapi.yaml & favicon.ico to the excludePaths, see above?

Reply:
No - The IP filter stopes anybody except those allowed from accessing anything. The only reason we allow the actuator calls through is for the health check. As mentioned we could restrict these to internal IP addresses  - and we might at some point.